### PR TITLE
Observability updates for 1.8 release

### PR DESCRIPTION
**Pull request by donatwork**
# Description
Updates to Observability versions shipping in CSM 1.8.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
|https://github.com/dell/csm/issues/885|

# Checklist:

- [ ] Have you run a grammar and spell checks against your submission?
- [X] Have you tested the changes locally?
- [ ] Have you tested whether the hyperlinks are working properly?
- [ ] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

File: content/docs/observability/deployment/offline.md [hunk 1]
```diff
@@ -250,7 +250,7 @@ To perform an offline installation of a Helm chart, the following steps should b
     kubectl get configmap powermax-config-params -n [CSI_DRIVER_NAMESPACE] -o yaml | sed 's/namespace: [CSI_DRIVER_NAMESPACE]/namespace: [CSM_NAMESPACE]/' | kubectl create -f -
     ```
 
-    If the CSI driver configmap name is not the default `powermax-config-params`, please use the following command to copy configmap:  
+    If the CSI driver configmap name is not the default `powermax-config-params`, please use the following command to copy configmap:
```
Comment (sharont58) [inline content/docs/observability/deployment/offline.md:253]: ... use the following command ...

(remove "please"

File: content/docs/observability/deployment/offline.md [hunk 2]
```diff
@@ -260,14 +260,14 @@ To perform an offline installation of a Helm chart, the following steps should b
     ```bash
 
     kubectl get secret karavi-authorization-config proxy-server-root-certificate proxy-authz-tokens -n [CSI_DRIVER_NAMESPACE] -o yaml | sed 's/namespace: [CSI_DRIVER_NAMESPACE]/namespace: [CSM_NAMESPACE]/' | sed 's/name: karavi-authorization-config/name: powermax-karavi-authorization-config/' | sed 's/name: proxy-server-root-certificate/name: powermax-proxy-server-root-certificate/' | sed 's/name: proxy-authz-tokens/name: powermax-proxy-authz-tokens/' | kubectl create -f -
-    ``` 
+    ```
 
-4. Now that the required images have been made available and the Helm chart's configuration updated with references to the internal registry location, installation can proceed by following the instructions that are documented within the Helm chart's repository.  
+4. Now that the required images have been made available and the Helm chart's configuration updated with references to the internal registry location, installation can proceed by following the instructions that are documented within the Helm chart's repository.
```
Comment (sharont58) [inline content/docs/observability/deployment/offline.md:265]: Suggestion:
After the images are available and the Helm chart configuration is updated, follow the instructions within the Helm chart's repository to complete the installation.

File: content/docs/observability/upgrade/_index.md [hunk 3]
```diff
@@ -106,27 +106,26 @@ These instructions can be followed when a Helm chart was installed and will be u
...
+   3. (Optional) Enable Karavi Observability for PowerFlex/PowerScale to use an existing instance of Karavi Authorization for accessing the REST API for the given storage systems.
+      **Note**: Assuming that if the Karavi Observability's Authorization has been enabled in the phase of [Offline Karavi Observability Helm Chart Installer](../deployment/offline), the Authorization Secrets/Configmap have been copied to the Karavi Observability namespace.
+      A sample configuration values.yaml file is located [here](https://github.com/dell/helm-charts/blob/main/charts/karavi-observability/values.yaml).
+      In your own configuration values.yaml, you need to enable PowerFlex/PowerScale Authorization, and provide the location of the sidecar-proxy Docker image and URL of the Karavi Authorization proxyHost address.
+
+   4. Now that the required images have been made available and the Helm chart's configuration updated with references to the internal registry location, installation can proceed by following the instructions that are documented within the Helm chart's repository.
```
Comment (sharont58) [inline content/docs/observability/upgrade/_index.md:117]: Suggestion:
After the images are available and the Helm chart configuration is updated, follow the instructions within the Helm chart's repository to complete the installation.

File: content/docs/observability/upgrade/_index.md [hunk 4]
```diff
@@ -106,27 +106,26 @@ These instructions can be followed when a Helm chart was installed and will be u
...
+      **Note**: Assuming that Your Secrets from CSI Drivers have been copied to the Karavi Observability namespace in the phase of [Offline Karavi Observability Helm Chart Installer](../deployment/offline)
```
Comment (sharont58) [inline content/docs/observability/upgrade/_index.md:118]: **Note**: Your Secrets from the CSI Drivers must be located in the Karavi Observability namespace in the phase of [Offline Karavi Observability Helm Chart Installer](../deployment/offline)

_Is "phase" the correct word in this sentence?_

**Review by sharont58 — DISMISSED** [review]
See comments

File: content/docs/observability/deployment/offline.md (hunk 1 above)
Comment (donatwork) [reply content/docs/observability/deployment/offline.md]: Done

File: content/docs/observability/deployment/online.md [hunk 5]
```diff
@@ -33,14 +33,14 @@ If the Authorization module is enabled for the CSI drivers installed in the same
...
-## Prerequisites 
+## Prerequisites
 
 - Helm 3.3
 - The deployment of one or more [supported](../#supported-csi-drivers) Dell CSI drivers
 
 ## Online Installer
 
-The following instructions can be followed to install CSM for Observability in an environment that has an internet connection and is capable of downloading the required Helm chart and Docker images.  
+The following instructions can be followed to install CSM for Observability in an environment that has an internet connection and is capable of downloading the required Helm chart and Docker images.
```
[the remark this answers is not in the document]
Comment (donatwork) [reply content/docs/observability/deployment/online.md]: Done

File: content/docs/observability/deployment/offline.md (hunk 2 above)
Comment (donatwork) [reply content/docs/observability/deployment/offline.md]: Done

File: content/docs/observability/upgrade/_index.md (hunk 4 above)
Comment (donatwork) [reply content/docs/observability/upgrade/_index.md]: Reworded.

File: content/docs/observability/deployment/offline.md [hunk 6]
```diff
@@ -106,15 +106,15 @@ To perform an offline installation of a Helm chart, the following steps should b
...
-      dellemc/csm-topology:v1.5.0 -> <my-registry>:5000/csm-topology:v1.5.0
-      dellemc/csm-metrics-powerflex:v1.5.0 -> <my-registry>:5000/csm-metrics-powerflex:v1.5.0
-      dellemc/csm-metrics-powerstore:v1.5.0 -> <my-registry>:5000/csm-metrics-powerstore:v1.5.0
-      dellemc/csm-metrics-powerscale:v1.2.0 -> <my-registry>:5000/csm-metrics-powerscale:v1.2.0
-      dellemc/csm-metrics-powermax:v1.0.0 -> <my-registry>:5000/csm-metrics-powerscale:v1.0.0
+      dellemc/csm-topology:v1.6.0 -> <my-registry>:5000/csm-topology:v1.6.0
+      dellemc/csm-metrics-powerflex:v1.6.0 -> <my-registry>:5000/csm-metrics-powerflex:v1.6.0
+      dellemc/csm-metrics-powerstore:v1.6.0 -> <my-registry>:5000/csm-metrics-powerstore:v1.6.0
+      dellemc/csm-metrics-powerscale:v1.3.0 -> <my-registry>:5000/csm-metrics-powerscale:v1.3.0
+      dellemc/csm-metrics-powermax:v1.1.0 -> <my-registry>:5000/csm-metrics-powerscale:v1.1.0
```
Comment (santhoshatdell) [inline content/docs/observability/deployment/offline.md:117]: On the right side, change powerscale to powermax.

**Review by santhoshatdell — DISMISSED** [review]

**Review by harshaatdell — DISMISSED** [review]

File: content/docs/observability/deployment/offline.md (hunk 6 above)
Comment (donatwork) [reply content/docs/observability/deployment/offline.md]: Fixed.

**Review by santhoshatdell — DISMISSED** [review]

**Review by anandrajak1 — DISMISSED** [review]

**Comment by santhoshatdell** [comment]
content/docs/deployment/csmoperator/_index.md needs changes. Please check.

**Comment by donatwork** [comment]
> content/docs/deployment/csmoperator/_index.md needs changes. Please check.

Observability 1.6 is listed for PowerScale which was the only 2.8 config. Seems like the update of that table should fall on the Operator team as it crosses multiple products. I added the missing rows for Observability and will leave the rest of the columns on the respective driver teams.

**Review by santhoshatdell — APPROVED** [review]

**Review by bharathsreekanth — APPROVED** [review]

**Pull request merged**